scripts/process/process-hydrograph.R

```diff
@@ -61,39 +60,46 @@ process.plot_hydrographTotal <- function(viz=getContentInfo("NMHydrograhTotal-sv
     rects <- xml_find_all(rect_svg_all, '//*[local-name()="rect"]')
     rect_svg <- rects[length(rects)]
     
-    xml_set_attr(rect_svg, 'id', paste0("b",yr))
+    xml_set_attr(rect_svg, 'data-year', as.character(yr))
```

Comment:
Nope, this is a direct request from @mbucknell . Apparently it's a schnazzy feature. I'll allow you two to discuss...